x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -1208,7 +1208,7 @@ public enum Cap {
         /**
          * Support knn function
          */
-        KNN_FUNCTION_V2(Build.current().isSnapshot()),
+        KNN_FUNCTION_V3(Build.current().isSnapshot()),
```

Comment:
Super nitpick - I wonder if we could create an alias that this could reference (so when we go to V4 we only need to change one variable when testing for test compatibility)

Reply:
Something like KNN_FUNCTION_CURRENT = EsqlCapabilities.KNN_FUNCTION_V3?

Reply:
Not sue if that works with CSV tests 🤔  . I'd say let's keep this as it's an established pattern in ES|QL. 

Maybe I'll give it a try for the next one and see what it looks like from a changes perspective 👍